docs/mp/jaxrs/10_reactive-routing.adoc

```diff
@@ -121,4 +121,4 @@ io.helidon.examples.AdminService:
     required: true
   routing-path:
     path: "/management"
-----
+----
```

Comment:
This looks like a white space change. This should be displayed in the diff as "File renamed without changes"